test/node/miscellaneous.js

```diff
@@ -18,6 +19,13 @@ describe('Miscellaneous Tests', () => {
         await client.logout();
     });
 
+    it('should create a conversation', async () => {
```

Comment:
"should create a conversation" should not be here

Reply:
done